app/views/cases/shared/_case_list.html.slim

```diff
@@ -52,7 +52,9 @@
             td aria-label="#{t('common.case_list.number')}"
               = case_link_with_hash(kase, :number, @page, position)
             td aria-label="#{t('common.case_list.flag')}"
-              = kase.highlight_flag
+              - if kase.highlight_flag != " "
```

Comment:
I don't think you want to change anything here because that change will apply to all case types. You should try to make use of the `trigger_case_marker` method that already formats the flag for trigger cases.